docs/blog/2022/2022-09-28-fable-4-theta.md

```diff
@@ -0,0 +1,76 @@
+
+## Language Status
+
+Managing versions with multiple language targets is... complicated. Should we hold Fable 4 release until all languages are considered stable? Should we split each language target into a different tool?
+
+After some discussion within the team, we have decided to keep a single codebase and tool to maximize the synergies that have been working so well. But on the other hand, we also want to release Fable 4 as soon as possible so then community can move on and we don't need to maintain both codebases for Fable 3 and 4. To solve the conundrum, we have decided to keep a different status for each language even after releasing Fable 4 "stable". You will see one of the following status when starting the tool depending on the selected language.
+
+- **stable**: The target is ready for production projects. Please report any bug you find.
+
+- **beta**: The target may lack some F#/FSharp.Core features but the output is not expected to change drastically, particularly in interop scenarios. Community feedback is crucial at this stage to iron out last issues and know where to focus our efforts.
+
+- **alpha**: We're committed to making the language a working target but there's still work to do and the representation of F# types and the way to interop with native code is still subject to change.
+
+- **experimental**: The target has been added by a contributor but it's not currently maintained and may be removed in the future. If you are interested please consider joining the team to maintain the target.
+
+With this in mind, at the time of writing this is the status for each of the language targets.
+
+- **JavaScript: beta** - We are aiming at no breaking changes for JS, but please note you may need to update some packages (see below). Please try upgrading your projects and let us know if you have any problem so we can fix it before the stable release.
+
+- **Python: beta** - Python compilation has been progressing for more than a year now. We believe it will be ready for v4 release but we encourage you to try it out already so we know what are the most interesting areas to explore for F# to Python compilation.
+
+- **Rust: alpha** - Rust compilation is already quite good, but there is still discussion going on for some compilation fundamentals, so there is still room for change.
+
+- **TypeScript: alpha** - We're moving TypeScript from its original experimental status to alpha, this means we want to make it a working target for Fable 4 in the future, but there's still work ahead to make sure compilation is on par with JS.
+
+- **PHP: experimental** - PHP compilation is not currently maintained.
+
+## Packages Updated for Fable 4
+
+A few dependencies needed updates to be compatible with Fable 4. If your project includes some of them, please be sure to use the versions listed below (or higher) before upgrading to Fable 4.
+
+- Thoth.Json v6.0
+- Fable.SimpleJson v3.24
+- Feliz v2.0.0-prerelease-003
```

Comment:
I think Fulma, Elmish v4 libraries also needs to be updated because of the breaking changes

Reply:
You're right. I was trying to distinguish the updates because of Fable 4 (the ReactComponent plugin) vs other updates. I'll try to make that clearer 👍